plugin/src/main/groovy/com/mendhak/gradlecrowdin/DownloadTranslationsTask.groovy

```diff
@@ -10,40 +10,49 @@ class DownloadTranslationsTask extends DefaultTask {
     def projectId
     def renameMapping
     def excludePattern
+    def branch
 
     @TaskAction
     def downloadTranslations() {
+        def branchEncoded = URLEncoder.encode((branch ?: '').replaceAll('[\\\\/:*?"<>|]', '_'), "UTF-8")
 
-        def buildSubDir = new File(project.buildDir.getPath() , "/crowdin-plugin")
+        def buildSubDir = new File(project.buildDir.getPath(), "/crowdin-plugin")
         buildSubDir.mkdirs()
 
         //Tell Crowdin to do an export
-        def exportUrl = sprintf('http://api.crowdin.net/api/project/%s/export?key=%s', [projectId, apiKey])
-        ant.get(src: exportUrl, dest: new File(buildSubDir.getPath(), "export.xml"), verbose:true)
+        def exportUrl = sprintf('https://api.crowdin.com/api/project/%s/export?key=%s', [projectId, apiKey])
+        if (branch != null) {
+            exportUrl += '&branch=' + branchEncoded
```

Comment:
OK just figured out why you're doing this.  You can't just leave `&branch=`, their API doesn't handle empty values here, how annoying!